### PR TITLE
Ensure folks know to enable dotfiles while troubleshooting

### PR DESCRIPTION
**Pull request by cdwort**
### Why:

Addresses support ticket question. We forgot to ask folks to check this setting!

### What's being changed (if available, include any code snippets, screenshots, or gifs):

Adds a step to check dotfiles config for Codespaces.

### Check off the following:

- [ ] I have reviewed my changes in staging (look for the "Automatically generated comment" and click the links in the "Preview" column to view your latest changes).
- [ ] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by cmwilson21** [comment]
@cdwort Thanks so much for opening a PR! I'll get this triaged for review :zap:

File: content/codespaces/troubleshooting/troubleshooting-dotfiles-for-codespaces.md [hunk 1]
```diff
@@ -14,9 +14,11 @@ shortTitle: Dotfiles
 If your codespace fails to pick up configuration settings from dotfiles, you should work through the following debugging steps.
 
 1. Make sure your dotfiles repository is public. If you have secrets or sensitive data you want to use in your codespace, use [Codespace secrets](/codespaces/managing-your-codespaces/managing-encrypted-secrets-for-your-codespaces) instead of private dotfiles.
-2. Check `/workspaces/.codespaces/.persistedshare/dotfiles` to see if your dotfiles were cloned.
+2. Enable dotfiles by checking "Automatically install dotfiles" in [your personal Codespaces settings](https://github.com/settings/codespaces).
```
Comment (hubwriter) [inline content/codespaces/troubleshooting/troubleshooting-dotfiles-for-codespaces.md:17]:
```suggestion
2. Enable dotfiles by selecting **Automatically install dotfiles** in [your personal Codespaces settings](https://github.com/settings/codespaces).
```

Comment (hubwriter) [reply content/codespaces/troubleshooting/troubleshooting-dotfiles-for-codespaces.md:17]: Small changes to our house style.

**Review by hubwriter — DISMISSED** [review]
Thanks @cdwort - that's a useful addition. 👍

**Comment by hubwriter** [comment]
I don't have upload access to https://github.com/cdwort/docs to add the new image file to the repo, so I'm going to move this to the internal repo so that I can finish this off.

**Pull request closed without merging**